tsfresh/feature_extraction/extraction.py

```diff
@@ -132,48 +123,38 @@ def extract_features(timeseries_container, default_fc_parameters=None,
 
     # Always use the standardized way of storing the data.
     # See the function normalize_input_to_internal_representation for more information.
-    kind_to_df_map, column_id, column_value = \
-        dataframe_functions.normalize_input_to_internal_representation(df_or_dict=timeseries_container,
-                                                                       column_id=column_id,
+    df_melt, column_id, column_kind, column_value = \
```

Comment:
`df_melt` contains the chunks, right?

maybe we can construct an object for the chunks (sorry for my java ;-) )